src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  height: 408px;
+
+  padding: 32px 16px 16px;
+
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+
+  background: #fff;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: var(--main-accent);
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    margin-top: 0;
+    font-size: 10px;
+    line-height: 14px;
+    color: var(--secondary);
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin: 0;
+    margin-bottom: 24px;
+  }
+
+  &__count-review {
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    margin-bottom: 0;
+    width: 53px;
+    height: 16px;
+    margin-top: 0;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    margin-top: 0;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    font-weight: 400;
```

Comment:
It's a default value for this rule, you can check it [HERE](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight)
Remove in all cases
```suggestion
```